mir-ci/mir_ci/robot_libraries/Screencopy.py

```diff
@@ -125,9 +125,9 @@ def _log_failed_match(self, screenshot, template):
         """Log a failure with template matching."""
         template_img = Image.open(template)
         template_string = (
-            'Template was:<br /><img src="data:image/png;base64,' f'{self._to_base64(template_img)}" /><br />'
+            'Template was:<br /><img width="100%" src="data:image/png;base64,' f'{self._to_base64(template_img)}" /><br />'
```

Comment:
```suggestion
            'Template was:<br /><img style="max-width: 100%" src="data:image/png;base64,' f'{self._to_base64(template_img)}" /><br />'
```

How about using `max-width`? It won't resize images that already fit, and it also won't cause overflow.